base_cron_exclusion/models/ir_cron.py

```diff
@@ -67,5 +67,5 @@ def _process_job(cls, db, cron_cr, job):
             res = super()._process_job(db, cron_cr, job)
         finally:
             locked_crons.close()
-            _logger.debug("released blocks for cron job %s" % job["cron_name"])
+            _logger.debug(f"released blocks for cron job {job['cron_name']}")
```

Comment:
Better to use the parameters of logger methods for the interpolation:
```suggestion
            _logger.debug("released blocks for cron job %s", job['cron_name'])
```

Reply:
Hello @sebalix, 
thank for your suggestions.